paper/paper.bib

```diff
@@ -0,0 +1,835 @@
+  abstract = "Blood oxygen-level-dependent functional magnetic resonance
+              imaging (BOLD fMRI) has rapidly become a popular technique for
+              the investigation of brain function in healthy individuals,
+              patients as well as in animal studies. However, the BOLD signal
+              arises from a complex mixture of neuronal, metabolic and vascular
+              processes, being therefore an indirect measure of neuronal
+              activity, which is further severely corrupted by multiple
+              non-neuronal fluctuations of instrumental, physiological or
+              subject-specific origin. This review aims to provide a
+              comprehensive summary of existing methods for cleaning the BOLD
+              fMRI signal. The description is given from a methodological point
+              of view, focusing on the operation of the different techniques in
+              addition to pointing out the advantages and limitations in their
+              application. Since motion-related and physiological noise
+              fluctuations are two of the main noise components of the signal,
+              techniques targeting their removal are primarily addressed,
+              including both data-driven approaches and using external
+              recordings. Data-driven approaches, which are less specific in
+              the assumed model and can simultaneously reduce multiple noise
+              fluctuations, are mainly based on data decomposition techniques
+              such as principal and independent component analysis.
+              Importantly, the usefulness of strategies that benefit from the
+              information available in the phase component of the signal, or in
+              multiple signal echoes is also highlighted. The use of global
+              signal regression for denoising is also addressed. Finally,
+              practical recommendations regarding the optimization of the
+              preprocessing pipeline for the purpose of denoising and future
+              venues of research are indicated. Through the review, we
+              summarize the importance of signal denoising as an essential step
+              in the analysis pipeline of task-based and resting state fMRI
+              studies.",
+  journal  = "Neuroimage",
+  volume   =  154,
+  pages    = "128--149",
+  month    =  jul,
+  year     =  2017,
+  keywords = "BOLD fMRI; Denoising methods; Motion artifacts; Multi-echo;
+              Phase-based methods; Physiological noise",
+  language = "en"
+}
+
+@UNPUBLISHED{asyraff2020stimulus,
+  title  =    "Stimulus-independent neural coding of event semantics:
+               Evidence from cross-sentence fMRI decoding",
+  author =    "Asyraff, Aliff and Lemarchand, Rafael and Tamm, Andres and Hoffman, Paul",
+  journal =   "bioRxiv",
+  year    =   2020,
+  publisher = "biorxiv.org"
+}
```

Comment:
I think we'll need to include a DOI here, which `UNPUBLISHED` doesn't support. So I'd update from `UNPUBLISHED` to `MISC` to render that field appropriately. Maybe :

```suggestion
@MISC{asyraff2020stimulus,
  title  =    "Stimulus-independent neural coding of event semantics:
               Evidence from cross-sentence fMRI decoding",
  author =    "Asyraff, Aliff and Lemarchand, Rafael and Tamm, Andres and Hoffman, Paul",
  journal =   "bioRxiv",
  year    =   2020,
  doi = {10.1101/2020.10.06.327817}
  url = {https://www.biorxiv.org/content/10.1101/2020.10.06.327817v1}
}
```